lib/python/picongpu/pypicongpu/simulation.py

```diff
@@ -55,7 +56,7 @@ class Simulation(RenderedObject):
     """
 
     # may not use util.build_typesafe_property since this attribute is usually never initialized
```

Comment:
How does this comment relate to the next line?

Reply:
good catch, left over from previous version, I removed the comment